### DISCUSSION TITLE
Windows <-> Mac Immich

### DISCUSSION THREAD

**Discussion by Tarun-Sreepada** [immich-app/immich#7575] [Q&A]
I have Immich on my Windows PC and I want to transfer it to Mac so I can view it on my Mac or Mac to Windows in case I have multiple files and I would like to leverage the Windows PC processing power.

I looked at the backup and restore (https://immich.app/docs/administration/backup-and-restore/) but I don't really understand what it says or if it is the right answer to solving my problem.

**Comment by mmomjian** [comment]
What is the exact issue you are trying to solve? Am I understanding that your goal is to frequently move Immich between different hosting servers running different operating systems? I would definitely not recommend that, while it is certainly possible to migrate an installation of Immich it is not designed to do this with any frequency. To view photos on your Mac, you should be using the web interface.

**Comment by Tarun-Sreepada** [comment]
Oh ok thanks.
